src/index.html

```diff
@@ -8,7 +8,224 @@
     <link rel="stylesheet" href="./css/index.scss" />
   </head>
   <body>
-    <div id="main"></div>
+    <div id="main">
+      <!--Header-->
+      <header>
+        <div>
+          <span>Home</span>
+          <span style="text-align: right;">Download My Full Resume</span>
```

Comment:
Please avoid using inline styles as it is not easy to maintain and be overwritten.

Besides, `text-align` that you applied cannot achieve the effect of pushing the download button to the right side.